Neos.Neos/Classes/Fusion/DimensionsMenuImplementation.php

```diff
@@ -209,4 +209,18 @@ protected function calculateTargetDimensionsForCombination(array $dimensionCombi
 
         return $targetDimensions;
     }
+
+    /**
+     * Returns TRUE if the node has a inaccessible parent.
+     *
+     * @param NodeInterface $node
+     * @return bool
+     */
+    protected function isNodeParentHidden(NodeInterface $node): bool
```

Comment:
The method name is a bit missleading, shouldn't it be named "areNodeParentsHidden"?

Reply:
I used the name as it’s only important whether one parent is hidden.

Are you sure your method return value definition is fine with Neos 3.3 and PHP7.0?